ext/standard/basic_functions.stub.php

```diff
@@ -2172,18 +2221,27 @@ function syslog(int $priority, string $message): true {} // TODO make return typ
 #endif
 
 #ifdef HAVE_INET_NTOP
-/** @refcount 1 */
+/**
+ * @compile-time-eval
+ * @refcount 1
+ */
```

Comment:
Same here for this one and the one below regarding the locale problem I mentioned in ip2long & long2ip.

Reply:
`inet_ntop` takes binary/packed IP format and converts it into ipv4/ipv6 address, I am not aware of any locale dependency

Reply:
In the links I sent you should take a look at the attributes section. It mentions "locale", which means that it can read from the locale (and probably do something with that info as otherwise it would be quite useless to read from it)

Reply:
resolving in favor of https://github.com/php/php-src/pull/10771#discussion_r1130059596 discussion